docker/docker.engine.yaml

```diff
@@ -171,3 +171,6 @@ auth:
       provider_id: zoom
       client_id: ${env:ZOOM_CLIENT_ID}
       client_secret: ${env:ZOOM_CLIENT_SECRET}
+
+dashboard:
+  redirect_uri: "http://localhost:9099/dashboard"
```

Comment:
Is there anything underlying that needs to be changed?

Reply:
Nope, this already works!